_docs_integrate/delete-a-relationship.md

```diff
@@ -0,0 +1,22 @@
+---
+# Start automatic generation
+permalink: integrate/delete-a-relationship
+published: false
+title: "Delete a Relationship"
```

Comment:
Should we accordingly rename this to "Delete Relationships" or "Terminate Relationships"? And do we want to get rid of this "Handling data deletion" category, since Attribute deletion is moved too, and rename the "Get in touch with other Identities" category to something more general about Relationships?